src/test/java/io/stargate/sgv2/jsonapi/service/schema/model/JsonapiTableMatcherTest.java

```diff
@@ -44,7 +65,7 @@ public void partitionColumnsTooMany() {
                   QueryOuterClass.ColumnSpec.newBuilder().setName("key2").build())
               .build();
 
-      boolean result = tableMatcher.test(table);
+      boolean result = tableMatcher.test(null);
```

Comment:
hmm wondering why this would be null? here and other cases below. I do see this class is disabled.

Reply:
@Hazel-Datastax ?

Reply:
All the data structures in `JsonapiTable` have been changed and I have to rewrite all the unit tests. To save time, I didn't do that and just replaced the `null` to avoid the compile error. I will fix that later with all the unit tests.

Reply:
Sure